prac2-pages/Siddharth_gautam_login_page/lib/other.dart

```diff
@@ -0,0 +1,210 @@
+  bool isEmail=true;
+  bool value=false;
+  bool changed=false;
+  String change='Don’t select this option if you’re using a public computer';
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(20),
+      child: ListView(
+        children: <Widget>[
+          Container(
+            alignment: Alignment.center,
+            child: Text(
+              'Login Verification',
+              style: TextStyle(
+                fontSize: 24,
+                fontWeight: FontWeight.bold,
+              ),
+            ),
+          ),
+          SizedBox(
+            height: 30,
+          ),
+          Container(
+            alignment: Alignment.center,
+            padding: EdgeInsets.fromLTRB(40, 0, 40,0),
+            child: Text(
+              'To continue logging in we need you to complete one of the following:',
+              textAlign: TextAlign.center,
+              style: TextStyle(
+                  fontSize: 18
+              ),
+            ),
+          ),
+          SizedBox(
+            height: 35,
+          ),
+          RadioListTile(
+            title:  Transform.translate(
+              offset: const Offset(-20, 0),
+              child:Text('Verification code via email',
+                style: TextStyle(
+                    fontSize: 18
+                ),
+              ),
+            ),
+            value: 'We’ll send code to your email that you can enter to verify your identity',
+            groupValue: radio,
+            activeColor: Colors.grey[800],
+            onChanged: (val){
+              setState(() {
+                radio=val!;
+                isEmail=true;
+              });
+            },
+          ),
+
+          Container(
+            padding: EdgeInsets.fromLTRB(56,0,50,0),
+            child: isEmail ?
+            Text('$radio',style: TextStyle(fontSize: 14,color: Colors.grey[600]),
+            ):SizedBox(),
+          ),
+
+
+
+
+
+          RadioListTile(
+            title: Transform.translate(
+              offset: const Offset(-20, 0),
+              child: Text('Verification code via SMS',
+                style: TextStyle(
+                    fontSize: 18
+                ),
+              ),
+            ),
+            value: 'We’ll send a code to your mobile phone that can enter to verify your identity.',
+            groupValue: radio,
+            activeColor: Colors.grey[800],
+            onChanged: (val){
+              setState(() {
+                radio=val!;
+                isEmail=false;
+              });
+            },
+          ),
+
+          Container(
+            padding: EdgeInsets.fromLTRB(56,0,50,0),
+            child: isEmail ?SizedBox():
+            Text('$radio',style: TextStyle(fontSize: 14,color: Colors.grey[600]),),
+          ),
+
+
+          SizedBox(
+            height: 30,
+          ),
+
+          Container(
```

Comment:
Try to make the button accomodate the text within it, i.e the text within should be in a single line, and not get wrapped. :thought_balloon: